src/monika/Array_Coding_Exam/ReturnSumOfTargetNumUsingArray.java

```diff
@@ -0,0 +1,35 @@
+package monika.Array_Coding_Exam;
+/*Coding Exam - 1 : 18th April 2021
+
+Program 1 : 
+Given an array of integers nums and an integer target, return indices of the two numbers
+ such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].*/
+public class ReturnSumOfTargetNumUsingArray {
+	
+	int[] getSumOfIndices(int[] arr, int targetNum) {
+		int[] arr1 = new int[2];
+		for(int index=0;index<arr.length;index++) {
+			for(int innerindex=1;innerindex<arr.length;innerindex++) {
```

Comment:
Inner for loop should from i+1, think on this else will connect for this on Skype